File: hard/19_laptop_rentals/laptop_rentals_test.py
import unittest
from laptop_rentals import laptopRentals

class TestProgram(unittest.TestCase):
    def test_case_1(self):
        input = [[0, 2], [1, 4], [4, 6], [0, 4], [7, 8], [9, 11], [3, 10]]
        expected = 3
        actual = laptopRentals(input)
        self.assertEqual(actual, expected)

File: hard/19_laptop_rentals/laptop_rentals.test.js
const laptopRentals = require('./laptop_rentals');

describe('TestProgram', () => {
  it('Test Case 1', () => {
    const input = [
      [0, 2],
      [1, 4],
      [4, 6],
      [0, 4],
      [7, 8],
      [9, 11],
      [3, 10],
    ];
    const expected = 3;
    const actual = laptopRentals(input);
    expect(actual).toEqual(expected);
  });
});
